terraform/gce/gce-init.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+
+sudo yum -y install cloud-init cloud-initramfs-tools
+sudo reboot
```

Comment:
as of right now, I am not sure the cloud-init packages _need_ to be installed via terraform remote-exec nor if the reboot is required. But, perhaps this is more future proof should we do more with cloud-init going forward?